content/en/blog/_posts/2020-06-30-SIG-Windows-Spotlight/index.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: blog
+title: "SIG-Windows Spotlight"
+date: 2020-06-30
+slug: sig-windows-spotlight-2020
+---
+
+# SIG-Windows Spotlight
+_This post tells the story of how Kubernetes contributors work together to provide a container orchestrator that works for both Linux and Windows._
+
+<img alt="Image of a computer with Kubernetes logo" width="30%" src="KubernetesComputer_transparent.png">
+
+Most people who are familiar with Kubernetes associated it with Linux. That connection makes sense, since Kubernetes ran on Linux from its very beginning. However, many teams and organizations working on adopting Kubernetes need the ability to orchestrate containers on Windows. Since the release of Docker and rise to popularity of containers, there have been efforts both from the community and from Microsoft itself to make container technology as accessible in Windows systems as it is in Unix-based systems.
+
+Within the Kubernetes community, those who are passionate about making Kubernetes accessible to the Windows community can find a home in the Windows Special Interest Group. To learn more about SIG-Windows and the future of Kubernetes on Windows, I spoke to co-chairs [Mark Rossetti](https://github.com/marosset)  and [Michael Michael](https://github.com/michmike) about the SIG's goals and how others can contribute.
+
+## Intro to Windows Containers & Kubernetes
+
+Kubernetes is the most popular tool for orchestrating container workloads, so to understand the Windows Special Interest Group (SIG) within the Kubernetes project, it's important to first understand what we mean when we talk about running containers on Windows.
+
+***
+_"When looking at Windows support in Kubernetes," says SIG (Special Interest Group) Co-chair Mark Rossetti, "many start drawing comparisons to Linux containers. Although some of the comparisons that highlight limitations are fair, it is important to distinguish between operational limitations and differences between the Windows and Linux operating systems. Windows containers run the Windows operating system and Linux containers run Linux."_
+***
+
+In essence, any "container" is simply a process being run on its host operating system, with some key tooling in place to isolate that process and its dependencies from the rest of the environment. The goal is to make that running process safely isolated, while taking up minimal resources from the system to perform that isolation. On Linux, the tooling used to isolate processes to create "containers" commonly boils down to cgroups and namespaces (among a few others), which are themselves tools built in to the Linux Kernel.
+
+<img alt="If dogs were processes: containerization would be like giving each dog their own resources like toys and food using cgroups, and isolating troublesome dogs using namespaces" width="40%" src="cgroupsNamespacesComboPic.png">
```

Comment:
nit: This alt attribute does not make sense as it repeats the same text as the following heading.

Reply:
Updated to describe the image more generally.

Reply:
Use control plane :+1: